File: cocos2D/Lib/Ads/AdColony/iOS/SampleApps/ObjC/AdColonyBasic/AdColonyBasic/ViewController.m
//
//  ViewController.m
//  AdColonyBasic
//
//  Created by Owain Moss on 1/13/15.
//  Copyright (c) 2015 AdColony. All rights reserved.
//

#import "ViewController.h"

#import <AdColony/AdColony.h>

@implementation ViewController

#pragma mark - UIViewController Overrides

- (UIStatusBarStyle)preferredStatusBarStyle {
    return UIStatusBarStyleLightContent;
}

- (UIInterfaceOrientationMask)supportedInterfaceOrientations {
    return UIInterfaceOrientationMaskAll;
}

- (BOOL)shouldAutorotate {
    return YES;
}

#pragma mark - Video Playback

- (IBAction)triggerVideo {
    [AdColony playVideoAdForZone:@"vz908f23a0cc6a49e6b1" withDelegate:nil];
}
@end

File: cocos2D/Lib/Ads/AdColony/iOS/SampleApps/Swift/AdColonyBasic/AdColonyBasic/ViewController.swift
//
//  ViewController.swift
//  AdColonyBasic
//
//  Created by Owain Moss on 1/12/15.
//  Copyright (c) 2015 AdColony. All rights reserved.
//

import UIKit
import Foundation

class ViewController: UIViewController
{
    
    @IBOutlet weak var background: UIImageView!
    
    //=============================================
    // MARK:- UIViewController Overrides
    //=============================================
    
    override func preferredStatusBarStyle() -> UIStatusBarStyle
    {
        return UIStatusBarStyle.LightContent
    }
    
    override func supportedInterfaceOrientations() -> UIInterfaceOrientationMask
    {
        return UIInterfaceOrientationMask.All
    }
    
    override func shouldAutorotate() -> Bool
    {
        return true
    }
    
    //=============================================
    // MARK:- AdColony
    //=============================================
    
    @IBAction func triggerVideo(sender: AnyObject)
    {
        AdColony.playVideoAdForZone("vzf8fb4670a60e4a139d01b5", withDelegate: nil)
    }
}
